### PR TITLE
Add UI Framework: Framevuerk

### PR DESCRIPTION
**Pull request by nainemom**
**Framevuerk** is a Fast, Responsive, Without Dependencies, Both Direction Support and **Configurable** (before run-time) UI Framework based on Vue.js. I've created a simple template using Framevuerk and add an option to select it in the `saofile.js`.

I didn't commit `npm run test` changes to `index.test.js.*` files. Maybe you should doing that?

Framevuerk docs and demo:
https://framevuerk.com

Framevuerk source on github:
https://github.com/framevuerk/framevuerk

### CODE REVIEW

File: saofile.js [hunk 1]
```diff
@@ -42,6 +42,7 @@ module.exports = {
       pageSize: 10,
       choices: [
         { name: 'None', value: 'none' },
+        { name: 'Framevuerk', value: 'framevuerk' },
```
Comment (clarkdo) [inline saofile.js]: Better to put as alphabetical order

File: README.md [hunk 2]
```diff
@@ -49,6 +49,7 @@ yarn create nuxt-app <my-project>
     - [Axios](https://github.com/nuxt-community/axios-module)
 3. Choose your favorite UI framework:
     - None (feel free to add one later)
+    - [Framevuerk](https://framevuerk.com)
```
Comment (clarkdo) [inline README.md]: Better to put as alphabetical order

File: template/nuxt/nuxt.config.js [hunk 3]
```diff
@@ -36,7 +36,7 @@ module.exports = {
     ],
     link: [
       { rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' }
-      <%_ if (ui === 'vuetify') { _%>,
+      <%_ if (ui === 'vuetify' || ui === 'framevuerk') { _%>,
```
Comment (clarkdo) [inline template/nuxt/nuxt.config.js]:
```suggestion
      <%_ if (['vuetify', 'framevuerk'].includes(ui)) { _%>,
```

File: README.md (hunk 2 above)
Comment (nainemom) [reply README.md]: UI frameworks mentioned in README.md file isn't ordered at all. Where should I put **framevuerk**?

Comment (clarkdo) [reply README.md]: OK, I'll do it sepratly

**Comment by clarkdo** [comment]
Could you please resolve the conflict ?

**Comment by nainemom** [comment]
done.
should I commit `npm run test` changes to `index.test.js.*` files?

**Comment by clarkdo** [comment]
No worries, I'll fix for you, thanks for the contribution!

**Comment by nainemom** [comment]
Thx for merging :)
But unfortunately, you miss mentioning **Framevuerk** in `README.md` file. Should I do special action for that?

**Pull request merged**